pkg/plugins/platform/utils.go

```diff
@@ -45,9 +45,14 @@ func GetApps() map[string][]App {
 				}
 
 				err = site.Init(approot)
-				if err == nil {
-					apps[platformType] = append(apps[platformType], site)
+				if err != nil {
+					// Cast 'site' from type App to type LocalApp, so we can manually enter AppConfig values.
+					site := site.(*LocalApp)
```

Comment:
The cast returns 2 args, the struct (if it exists) and "ok" that tells you whether it existed. I'd rather see 
```
siteStruct, ok := site.(*LocalApp)
                                      if (ok) {
                                              siteStruct.AppConfig.Name = siteName
                                              siteStruct.AppConfig.AppType = appType
                                      }
```
There are two issues here:
1. Catch the case where the cast doesn't work (unlikely, but defensive code is good)
2. Don't change the meaning of the variable "site" in the middle of the code. It *was* one thing, and changing it to something completely different could throw somebody working on this in the future. Never forget that this has a life after you work on it.